src/transactions/test/InvokeHostFunctionTests.cpp

```diff
@@ -602,6 +602,31 @@ TEST_CASE("contract storage", "[tx][soroban]")
             true, type);
     };
 
+    auto bumpOp = [&](uint32_t bumpAmount,
+                      xdr::xvector<LedgerKey> const& readOnly) {
+        Operation bumpOp;
+        bumpOp.body.type(BUMP_FOOTPRINT_EXPIRATION);
+        bumpOp.body.bumpFootprintExpirationOp().ledgersToExpire() = bumpAmount;
+
+        SorobanResources bumpResources;
+        bumpResources.footprint.readOnly = readOnly;
+        bumpResources.instructions = 200'000;
+        bumpResources.readBytes = 1000;
```

Comment:
The R/W discrepancy seems weird, these should be equal, right?

Reply:
Yeah this was a copy paste. Fixed

Reply:
@dmkozh Based on what we discussed, shouldn't writeBytes be 0?